mmv1/third_party/terraform/tests/resource_bigquery_table_test.go

```diff
@@ -1050,10 +1098,10 @@ var testUnitBigQueryDataTableIsChangableTestCases = []testUnitBigQueryDataTableJ
 		changeable: false,
 	},
 	{
-		name:       "typeModeOmission",
+		name:       "modeToDefaultNullable",
 		jsonOld:    "[{\"name\": \"someValue\", \"type\" : \"BOOLEAN\", \"mode\" : \"REQUIRED\", \"description\" : \"someVal\" }]",
 		jsonNew:    "[{\"name\": \"someValue\", \"type\" : \"BOOLEAN\", \"description\" : \"some new value\" }]",
-		changeable: false,
+		changeable: true,
```

Comment:
If mode is omitted, it will be NULLABLE, so I believe this should match typeModeReqToNull (a few cases up) and be changeable. Previously it seems this was not working correctly?